code/modules/clothing/head/jobs.dm

```diff
@@ -145,6 +145,21 @@
 	strip_delay = 60
 	dog_fashion = /datum/dog_fashion/head/warden
 
+/obj/item/clothing/head/warden/drill
+	name = "warden's drill hat"
+	desc = "A special armored campaign hat with the security insignia emblazoned on it. Uses reinforced fabric to offer sufficient protection. Has the letters 'FMJ' enscribed on it's side."
+	icon_state = "wardendrill"
+	item_state= "wardendrill"
+	dog_fashion = null
+	var/datum/mutation/human/yelling
+
+/obj/item/clothing/head/warden/drill/equipped(mob/living/carbon/human/user, slot)
+	..()
+	if(slot == SLOT_HEAD)
+		user.dna.add_mutation(YELLING)
+	else

```

Comment:
Replace the "else" part with `dropped(mob/user)`, else it won't remove the mutation if the hat gets destroyed by acid etc.
Krav maga gloves have similar code that you could use.

Reply:
To clarify, the entire proc path is required.
```DM
/obj/item/clothing/head/warden/drill/dropped(mob/user)
```